core/src/edu/asu/c3simulator/screens/EmploymentScreen.java

```diff
@@ -0,0 +1,315 @@
+package edu.asu.c3simulator.screens;
+
+import java.util.LinkedList;
+import java.util.List;
+
+import com.badlogic.gdx.Game;
+import com.badlogic.gdx.Gdx;
+import com.badlogic.gdx.Screen;
+import com.badlogic.gdx.graphics.GL20;
+import com.badlogic.gdx.scenes.scene2d.Actor;
+import com.badlogic.gdx.scenes.scene2d.InputEvent;
+import com.badlogic.gdx.scenes.scene2d.Stage;
+import com.badlogic.gdx.scenes.scene2d.ui.Label;
+import com.badlogic.gdx.scenes.scene2d.ui.ScrollPane;
+import com.badlogic.gdx.scenes.scene2d.ui.Skin;
+import com.badlogic.gdx.scenes.scene2d.ui.Table;
+import com.badlogic.gdx.scenes.scene2d.ui.TextField;
+import com.badlogic.gdx.scenes.scene2d.ui.VerticalGroup;
+import com.badlogic.gdx.scenes.scene2d.utils.Align;
+import com.badlogic.gdx.scenes.scene2d.utils.ClickListener;
+import com.badlogic.gdx.utils.viewport.StretchViewport;
+import com.badlogic.gdx.utils.viewport.Viewport;
+import edu.asu.c3simulator.simulation.Company;
+import edu.asu.c3simulator.simulation.Employee;
+
+/**
+ * @author Krogstad, Nick
+ * Framework: Moore, Zachary 
+ */
+public class EmploymentScreen implements Screen
+{
+	
+	private class EmployeeListener extends ClickListener
+	{
+		private Employee employee;
+		
+		public EmployeeListener(Employee passedEmployee)
+		{
+			employee = passedEmployee;
+		}
+		
+		@Override
+		public void clicked(InputEvent event, float x, float y)
+		{
+			selectedEmployee = this.employee;
+			employeeName.setText("Name: " + employee.getEmployeeName());
+			employeePosition.setText("Position: " + employee.getEmployeePosition());
+			employeePayField.setText("" + employee.getHourlyWage());
+			employeePref.setText("Income Preference: $"
+					+ employee.getPreferredHourlyWage() + " / hr");
+			employeeMorale.setText("Morale: " + employee.getMorale() + " / 10");
+			netSalary.setText("Net Salary: " + employee.getNetSalary());
+			netSalary.setText("Averale Annual Bonus: " + employee.getAverageAnnualBonus()
+					+ "%");
+			netSalary.setText("Average Annual Raise: " + employee.getAverageAnnualRaise()
+					+ "%");
+			netBonuses.setText("Net Bonuses: " + employee.getNetBonuses());
+			employeePayTable.setVisible(true);
+		}
+	}
+	
+	private static final int DESIGN_WIDTH = 1280;
+	private static final int DESIGN_HEIGHT = 720;
+	private static final int DESIGN_SCREEN_CENTER_X = DESIGN_WIDTH / 2;
+	private static final int DESIGN_SCREEN_CENTER_Y = DESIGN_HEIGHT / 2;
+	private static final int ROSTER_OFFSET = 500;
+	private Label employeeName;
+	private Label employeePosition;
+	private Actor employeePay;
+	private Label employeePref;
+	private Label employeeMorale;
+	private Label netSalary;
+	private Label averageAnnualBonus;
+	private Label averageAnnualRaise;
+	private Label netBonuses;
+	
+	private Game game;
+	private Skin skin;
+	private Stage stage;
+	private TextField employeePayField;
+	private Employee selectedEmployee;
+	private Table employeePayTable;
```

Comment:
This table and employeePayField look like they should be grouped with the other labels above.